Adding initial contents (#1)
pyviz/website: doc/index.md

# How to visualize data in Python

Welcome to PyViz!  PyViz is a coordinated effort to make data visualization in Python easier to use, easier to learn, and more powerful.

Focusing on interactive plotting in web browsers, PyViz contains:

 - [Overviews](overviews/index.html) of the visualization landscape in Python and core concepts that many APIs build on.
 - [Tutorials](tutorials/index.html) showing how to use the available tools together to do a wide range of different tasks.
 - [Topics](htts://examples.pyviz.org) presenting narrative examples of generating visualizations for particular topics.

## This site

> We are actively building this site and are looking for feedback/ideas/contributions.

Reviewer: ```suggestion
If you are a part of the Python data visualization landscape, then this is _your_ site!  All content has been contributed by individual library authors and users, and you could be next!
```

> So if you are a maintainer, contributor, or power-user of a

Reviewer: ```suggestion
```

> python visualization tool, we want to hear from you! Please open an issue or

Reviewer: ```suggestion
Please open an issue or
```

PR on [this GitHub repo](https://github.com/pyviz/pyviz) to start a conversation.

> **NOTE:** The material that used to be hosted at https://pyviz.org can now be found at https://metaplot.org.

Reviewer: ```suggestion
**NOTE:** Some of the material previously hosted under Tutorials on this site is now hosted by metaplot.org and is linked from the "Metaplot" section of the Tutorials heading on this site.
```
